### PR TITLE
[Feature] Add SassDoc integration

### LINKED ISSUE #6: [Improvement] Documention: SassDoc
Utilizing [SassDoc](http://sassdoc.com/), especially mixins and functions can be documentated with annotations:
- examples
- parameter and return values with types and description
- private / public access
...

The documentation can be auto-generated that includes examples, references, requirements etc. ([see SassDoc example for Sass MQ](http://sass-mq.github.io/sass-mq/)).

### PR DESCRIPTION
**Pull request by csshugs**
Netlify deployment already set up. You can check out current build [here](https://fluidms.netlify.com/). For local testing, you can run `$ npm run sassdoc` and check out the compiled HTML under `dist/`.

See issue #6 

Closes #6

### CODE REVIEW

File: src/tools/_tools.font-size.scss [hunk 1]
```diff
@@ -2,6 +2,57 @@
 //     #FONT-SIZE
 // \*------------------------------------*/
 
+/// Generates a font-size declaration and (optionally) an equivalent
+/// line-height.
+///
+/// @param {number | length } $ms [0] - Determine the font-size by using a value
+///   of the modular scale.
+/// @param {number | auto | "normal" | "inherit" | "none" | "false"} $line-height
```
Comment (fuhlig) [inline src/tools/_tools.font-size.scss]: `{number | string}`

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "server": "npx http-server test/",
-    "sass:watch": "npx node-sass test/style.scss test/style.css --watch"
+    "sass:watch": "npx node-sass test/style.scss test/style.css --watch",
+    "sassdoc": "rm -rf dist/ && npx sassdoc src -d dist"
```
Comment (fuhlig) [inline package.json]: `rm` is a bash command, so it's not does not work cross-platform - except you use e.g. git bash on windows.

IMO there are a couple of options
1) leave as is, add a notice in the documentation to either use a proper cli or manually delete it
2) use a cross-platform remove tool like [rimraf](https://www.npmjs.com/package/rimraf) or [shelljs](https://github.com/shelljs/shx)
3) do not clean `dist/`. "worst" thing that could happen is leftover assets 

1. People might run into initial troubles
2. Easiest, but added dependency
3. Not clean (haha) but does not break anything

File: src/settings/_settings.config.scss [hunk 3]
```diff
@@ -6,29 +6,81 @@
...
+///           3,
+///           2,
+///           1,
+///           0,
+///           -1,
+///       ),
+///   ) !default;
+///
+/// @type map
 $FLUIDMS-GLOBAL-CONFIG: (
```
Comment (fuhlig) [inline src/settings/_settings.config.scss]: Not really part of this PR, but I'd recommend to omit the `GLOBAL` from the variable.

Some frameworks / libs use `global` or `base` to indicate a global setting / constant, but by utilizing the uppercase convention for global variables, this is obsolete and makes the variable more verbose

`$FLUIDMS-GLOBAL-CONFIG` => `$FLUIDMS-CONFIG`

**Review by fuhlig — CHANGES_REQUESTED** [review]
Great setup, just minor requests

File: package.json (hunk 2 above)
Comment (csshugs) [reply package.json]: Pardon my ignorance towards non-UNIX people.

I think 2. is the most reasonable option.

File: src/settings/_settings.config.scss (hunk 3 above)
Comment (csshugs) [reply src/settings/_settings.config.scss]: Good point. Although not part of this feature, I will change that in this PR anyway.

File: src/tools/_tools.font-size.scss (hunk 1 above)
Comment (csshugs) [reply src/tools/_tools.font-size.scss]: `string` as a param wouldn’t be correct, as that would create the impression that every string could be used, where in fact only these specific keywords are allowed. Any other value would provoke a Sass error.

Comment (fuhlig) [reply src/tools/_tools.font-size.scss]: They are still strings. You can add the allowed keywords in the description

Comment (csshugs) [reply src/tools/_tools.font-size.scss]: Copy that.

**Review by fuhlig — APPROVED** [review]
